packages/governance/test/swingsetTests/committeeBinary/bootstrap.js

```diff
@@ -4,50 +4,77 @@ import { E } from '@agoric/eventual-send';
 import { Far } from '@agoric/marshal';
 import buildManualTimer from '@agoric/zoe/tools/manualTimer.js';
 
+import {
+  ChoiceMethod,
+  QuorumRule,
+  ElectionType,
+  looksLikeQuestionSpec,
+} from '../../../src/question.js';
+
+const { quote: q } = assert;
+
 const makeVoterVat = async (log, vats, zoe) => {
   const voterCreator = E(vats.voter).build(zoe);
   log(`=> voter vat is set up`);
   return voterCreator;
 };
 
-async function addQuestion(qDetails, closingTime, tools) {
+const createQuestion = async (qDetails, closingTime, tools, quorumRule) => {
   const { registrarFacet, installations } = tools;
-  const { question, positions } = qDetails;
+  const { issue, positions, electionType } = qDetails;
   const closingRule = {
     timer: tools.timer,
     deadline: 3n,
```

Comment:
The `closingTime` arg doesn't seem to get used. Did you mean to use it here?

```suggestion
    deadline: closingTime,
```

Reply:
yep. done